content/en/tracing/profiler/search_profiles.md

```diff
@@ -174,6 +176,17 @@ CPU
 Wall Time
 : Shows the elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running.
 
+{{< /programming-lang >}}
+{{< programming-lang lang="nodejs" >}}
+
+Once enabled, the following profile types are collected:
+
+Wall Time
+: Shows the elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running.
```

Comment:
It seems a little odd to say that a function is _running_ if it is sleeping, but a walltime profiler would in fact attribute sleeps.

Is it more precise/comprehensible to say that the "function context is active?"  Something like that?

Total nit, please ignore if nothing good comes to mind.

Reply:
"Function context is active" seems a bit complex. I'm open to other ideas but don't have anything myself yet.

Reply:
Thanks for the suggestion :-) It could be doing things other than those 2 too.

Reply:
Just swap "function" for "application" maybe? 🤔

Reply:
Thanks for the idea :-) That wouldn't be quite the same thing if I understand you correctly. It's saying the width of each frame/box/function shown on the flame graph is based on how much time happened while that specific function was "running" (for various values of running).